src/index.ts

```diff
@@ -51,6 +52,9 @@ async function init(): Promise<void> {
     }
   }
   const promMiddleware = isProdEnv ? createPrometheusMiddleware() : undefined;
+  if ('pool' in db) {
```

Comment:
`// TODO: use "if (db instanceof PgDataStore)"`

Reply:
This is fixed.